### ISSUE TITLE
Implement `starknet.go` support for contract class endpoints

### ISSUE THREAD

**Issue by jelilat** [NethermindEth/rpc-request-builder#9]
Add autogenerated `starknet.go` code for the following RPC endpoints

- `starknet_getClass`
- `starknet_getClassHashAt`
- `starknet_getClassAt`

**Additional Context:**

- https://github.com/NethermindEth/rpc-request-builder/issues/1
- https://github.com/NethermindEth/rpc-request-builder/blob/main/src/api/rpcspec/methods.ts

**Comment by dhirajs0** [comment]
Hi @jelilat I would like to take this issue, please assign it to me.

**Comment by intldds** [comment]
Hi @dhirajs0, have you started working on this issue? 

In case you haven't, I have already completed it and have a PR ready to send.

**Comment by dhirajs0** [comment]
Hi @intldds , I have already implemented two APIs , one is left. I would suggest to please show interest before completing any issue, so that we could avoid duplication of efforts.  The is one more issue, not taken by anyone yet, please have a look.

**Comment by intldds** [comment]
Yeah, I've just commented because this issue is not assigned to anyone yet and from what I can see you haven't started yet @dhirajs0 -- but feel free to do it then :)

**Comment by dhirajs0** [comment]
> Yeah, I've just commented because this issue is not assigned to anyone yet and I was quick to finish it @dhirajs0 -- but feel free to do it then :)

Thank you :)  @intldds

**Comment by g4titan** [comment]
hello @jelilat, assign this task to me
